presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -77,16 +90,16 @@
 {
     private static final Logger log = Logger.get(BaseJdbcClient.class);
 
-    private static final Map<Type, String> SQL_TYPES = ImmutableMap.<Type, String>builder()
-            .put(BOOLEAN, "boolean")
-            .put(BIGINT, "bigint")
-            .put(INTEGER, "integer")
-            .put(SMALLINT, "smallint")
-            .put(TINYINT, "tinyint")
-            .put(DOUBLE, "double precision")
-            .put(REAL, "real")
-            .put(VARBINARY, "varbinary")
-            .put(DATE, "date")
+    private static final Map<Type, WriteMapping> WRITE_MAPPINGS = ImmutableMap.<Type, WriteMapping>builder()
+            .put(BOOLEAN, WriteMapping.booleanMapping("boolean", booleanWriteFunction()))
```

Comment:
Static import?

I also noticed the read mapping is `booleanReadMapping` but this is just `booleanMapping` (not `booleanWriteMapping`).

Reply:
After "Rename ReadMapping to ColumnMapping", `ColumnMapping` factory method follow `booleanMapping` naming pattern. 

I thought with these various mappings (ColumnMapping and WriteMapping) i don't want to static-import their factory methods, so i made the names shorter, as appropriate.